src/app/app.component.ts

```diff
@@ -39,6 +44,15 @@ export class AppComponent implements OnInit, OnDestroy {
     this.isLoggedIn$ = this.trackLoginService.isLoggedIn$;
     this.dismissedLatestTOS$ = this.tosBannerQuery.dismissedLatestTOS$;
     this.dismissedLatestPrivacyPolicy$ = this.tosBannerQuery.dismissedLatestPrivacyPolicy$;
+    this.displayLoggedInTOSBanner$ = this.tosBannerQuery.displayLoggedInTOSBanner$;
+    this.userQuery.user$.pipe().subscribe((user) => {
```

Comment:
It's the app object, so it may not matter that much, but I think you need to unsubscribe (or `takeUntil(....`).

Reply:
I had it before, but Gary explained it was pointless https://github.com/dockstore/dockstore-ui2/pull/1318#discussion_r678389800

But yes, you are reading it correctly. I'm assuming it's okay because Nneka reviewed the documentation that describes this and approved it.